test/api-digester/Outputs/Cake-abi.txt

```diff
@@ -29,3 +29,11 @@ cake1: Func S1.foo3() is now static
 cake1: Struct C6 is now with @_fixed_layout
 cake1: Var C1.CIIns1 changes from weak to strong
 cake1: Var C1.CIIns2 changes from strong to weak
+
+/* Fixed-layout Type changes */
+cake1: EnumElement FrozenKind.Fixed in a fixed layout type changes position from 1 to 2
+cake1: EnumElement FrozenKind.Rigid in a fixed layout type changes position from 2 to 1
+cake1: Func fixedLayoutStruct.foo() in a fixed layout type changes position from 0 to 1
```

Comment:
Only re-ordering stored instance properties is bad.

Re-ordering anything else (computed properties, static properties, methods, etc) is OK.

Reply:
is `VAD->hasStorage() && !VAD->isStatic()` sufficient to check this?  frozen enums don't need this constraint, right?

Reply:
Yes, it is sufficient. For enums, the equivalent is to just check all EnumElementDecls.